server/app/services/scheduled_nps_report_service.py

```
@@ -0,0 +1,347 @@
+        })
+
+        return True
+
+    @staticmethod
+    async def get_due_reports(db: AsyncSession) -> List[ScheduledNPSReport]:
+        """Get all scheduled reports that are due for execution."""
+        current_time = datetime.now()
+        
+        query = select(ScheduledNPSReport).where(
+            and_(
+                ScheduledNPSReport.is_active == True,
```

Ruff (0.13.1)

Avoid equality comparisons to `True`; use `ScheduledNPSReport.is_active:` for truth checks

Replace with `ScheduledNPSReport.is_active`

(E712)